### DISCUSSION TITLE
Getting started with examples

### DISCUSSION THREAD

**Discussion by doctorfree** [pomdtr/sunbeam#61] [Q&A]
Hi, I'm having a hard time getting started with the `sunbeam` examples. I just today downloaded `sunbeam` so perhaps I am missing something basic.

When I run any of the examples' installation commands I get the same error:

```
Error: could not refresh commands: unable to extract script metadata: no metadata found
```

For example, the following two commands produce this error:

```bash
sunbeam command add devdocs https://raw.githubusercontent.com/pomdtr/sunbeam/main/docs/examples/devdocs/devdocs.sh
```

```bash
sunbeam command add github https://raw.githubusercontent.com/pomdtr/sunbeam/main/docs/examples/github/github.sh
```

Do I need to add metadata somewhere to add any of the example commands?

Thanks for any help

**Comment by pomdtr** [comment]
Sorry the latest release might not be up to date with the docs. I'm working on a new extension system, which should be released soon.

Can you hold off usage for 1 week ?

**Reply by doctorfree** [reply]
Yes, I can hold off usage, I have a large todo list and checking out `sunbeam` does not block anything else. Take as much time as you need.

Thanks for the prompt response, I can move on to another task.
